src/main/java/com/codessquad/qna/web/AnswerController.java

```diff
@@ -0,0 +1,34 @@
+package com.codessquad.qna.web;
+
+import com.codessquad.qna.domain.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import javax.servlet.http.HttpSession;
+
+@Controller
+@RequestMapping("/questions/{questionId}/answers")
+public class AnswerController {
+
+    @Autowired
+    private AnswerRepository answerRepository;
+
+    @Autowired
+    private QuestionRepository questionRepository;
+
+    @PostMapping("")
+    public String create(@PathVariable Long questionId, String contents, HttpSession session) {
+        if (!HttpSessionUtils.isLoginUser(session)) {
+            return "/users/login";
+        }
+
+        User sessionedUser = HttpSessionUtils.getUserFromSession(session);
+        Question selectedQuestion = questionRepository.findById(questionId).get();
+        Answer answer = new Answer(sessionedUser, selectedQuestion, contents);
+        answerRepository.save(answer);
+        return String.format("redirect:/questions/%d", questionId);
```

Comment:
컨트롤러에서 리턴으로 나가는 view name으로는 문자열 덧셈 연산이 더 가독성이 좋습니다.

```suggestion
        return "redirect:/questions/" + questionId;